pkg/test/echo/cmd/server/main.go

```diff
@@ -74,12 +75,17 @@ var (
 				portIndex++
 			}
 
+			bindPodIPPortsMap := make(map[int]bool, len(bindPodIPPorts))
+			for _, port := range bindPodIPPorts {
+				bindPodIPPortsMap[port] = true
+			}
 			s := server.New(server.Config{
-				Ports:     ports,
-				TLSCert:   crt,
-				TLSKey:    key,
-				Version:   version,
-				UDSServer: uds,
+				Ports:             ports,
+				TLSCert:           crt,
+				TLSKey:            key,
+				Version:           version,
+				UDSServer:         uds,
+				BindPodIPPortsMap: bindPodIPPortsMap,
```

Comment:
nit: this app can run outside of kubernetes, so we shouldn't use "pod"

Reply:
done